starter-code/server.js

```diff
@@ -66,18 +83,28 @@ app.put('/articles/:id', function(request, response) {
   // an author_id property, so we can reference it from the request.body.
   // TODO: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors(author, "authorUrl")
+     VALUES ($1, $2)
+     WHERE author = $1`,
+    [request.body.author,
+      request.body.authorUrl]
   )
```

Comment:
Well, after looking in this fro a little longer I see that this is garbage that does nothing...   Thank you!  I keep trying to understand and implement it.